### ISSUE TITLE
Bad torrent generated when only one file is in torrent

### ISSUE THREAD

**Issue by xuset** [xuset/planktos#11]
If only one file is passed to the planktos cli then a bad torrent is generated. The `files` section of the bad torrent is something like:

```
  "files": [
    {
      "path": "planktos",
      "name": "planktos",
      "length": <length>,
      "offset": 0
    }
  ]
```
When it should be:

```
  "files": [
    {
      "path": "planktos/<sha256 sum>",
      "name": "<sha256 sum>",
      "length": <length>,
      "offset": 0
    }
  ]
```

This discrepancy is likely releated to create-torrent treating the creation differently when it is a single or multi-file torrent

**Comment by georgeaf99** [comment]
How does one access the files section of the torrent?

**Comment by xuset** [comment]
I use the parse-torrent package which comes with a cli. `parse-torrent <file.torrent>` will print out the json version of the torrent file

**Comment by xuset** [comment]
Closed in #20
